source/table_view.tsx

```diff
@@ -101,13 +103,7 @@ export class TableView extends React.Component<Properties, State> implements
           {this.props.labels[i]}
         </th>);
     }
-    const startRow = (() => {
-      if(this.state.topRow > 0) {
-        return this.state.topRow - 1;
-      } else {
-        return 0;
-      }
-    })();
+    const startRow = Math.max(0, this.state.topRow - 1);
     const endRow = (() => {
       if(this.props.model.rowCount === 0) {
         return 0;
```

Comment:
Apply a similar principle to ```endRow``` as to ```startRow``` where you use an expression oriented approach instead of an imperative approach.